src/PrestaShopBundle/Resources/views/Admin/TwigTemplateForm/bootstrap_4_horizontal_layout.html.twig

```diff
@@ -110,6 +110,29 @@
 </div>
 {% endblock text_with_unit_widget %}
 
+{% block integer_with_unit_widget %}
```

Comment:
If the modification aims at the recent form only the widget in `prestashop_ui_kit_base.html.twig` should be enough But it doesn't hurt to add this in those form themes as well Even if they will be removed someday